sky/utils/kubernetes/deploy_remote_cluster.sh

```diff
@@ -247,12 +293,22 @@ echo "Cluster deployment completed. You can now run 'kubectl get nodes' to verif
 if [ "$INSTALL_GPU" == "true" ]; then
     echo -e "${YELLOW}GPU detected in the cluster. Installing Nvidia GPU Operator...${NC}"
     run_remote "$HEAD_NODE" "
+        # Create temporary askpass script
+        ASKPASS_SCRIPT=\$(mktemp)
+        trap 'rm -f \$ASKPASS_SCRIPT' EXIT INT TERM ERR QUIT
+cat > \$ASKPASS_SCRIPT << EOF
+#!/bin/bash
+echo "$PASSWORD"
+EOF
+        chmod 700 \$ASKPASS_SCRIPT
+        # Use askpass
+        export SUDO_ASKPASS=\$ASKPASS_SCRIPT
```

Comment:
This code snippet is used pretty often in this file, can we have a macro/function that can be reused instead of copy pasting?

Reply:
Yes, sorry for this.